gradle/validation/owasp-dependency-check.gradle

```diff
@@ -26,9 +26,12 @@ configure(rootProject) {
   dependencyCheck {
     failBuildOnCVSS = propertyOrDefault("validation.owasp.threshold", 7) as Integer
     formats = ['ALL']
-    skipProjects = [':solr:solr-ref-guide']
+    skipProjects = [':solr:solr-ref-guide', ':solr-missing-doclet']
     skipConfigurations = ['unifiedClasspath', 'permitUnusedDeclared']
     suppressionFile = file("${resources}/exclusions.xml")
+    analyzers {
+      assemblyEnabled = false
+    }
```

Comment:
I have some .Net projects locally and for some reason dependency check is picking this up and trying to scan the project. Since we don't use .Net - we can disable the .Net Assembly scanner. See details here: https://jeremylong.github.io/DependencyCheck/dependency-check-gradle/configuration.html